.circleci/config.yml

```diff
@@ -1030,12 +1027,14 @@ workflows:
                   requires:
                       - test
                   pre-steps:
-                      - secrethub/env-export:
-                            secret-path: graviteeio/cicd/azure/docker-registries/graviteeio/username
-                            var-name: AZURE_DOCKER_REGISTRY_USERNAME
-                      - secrethub/env-export:
-                            secret-path: graviteeio/cicd/azure/docker-registries/graviteeio/password
-                            var-name: AZURE_DOCKER_REGISTRY_PASSWORD
+                    - attach_workspace:
+                        at: /tmp
```

Comment:
Attaching the workspace here is a super bad idea as it will need to download the whole workspace (cost in machine time + network) just to read 2 files.